### PR TITLE
add user personas

### PR DESCRIPTION
**Pull request by Mana12011207**
Please review user personas and descriptions.

### CODE REVIEW

**Review by irenenguyen1017 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -45,8 +45,17 @@ The app targets shoppers across Australia who are looking for unique and quality
 - Japanese food products lovers
 
 ## User Stories 
+We construct user stories and personas to improve the online store's user experience for Japanese food products. The process involves identifying and embodying potential users' unique needs and expectations, including a system administrator at Sakura company.
 
-Working in progress...
+A problem Shizuka faces is the difficulty travelling to supermarkets where authentic Japanese food products are available, even with her three children. With the demands of raisingtrello  her children, she also needs to save time when shopping for her daily food. She is particularly concerned about the security of websites, especially when shopping online, and prefers to use sites that offer a secure login facility.
```
Comment (ellenpham) [inline README.md]: This sentence is not very concise, maybe changed from..

`A problem Shizuka faces is the difficulty travelling to supermarkets where authentic Japanese food products are available, even with her three children. With the demands of raisingtrello  her children, she also needs to save time when shopping for her daily food.`

to

"Shizuka, a mom of three, is facing the difficulty of travelling to supermarkets in a daily basis to shop for authentic Japanese food products. She needs a online grocery website that is user-friendly and easy navigated where she can browse and shop online when the kids are taking a nap and get the products delivered to home without the hassle of going to the physical shops."


Also, maybe change this term `secure login facility` to "secure login feature"

File: README.md [hunk 2]
```diff
@@ -45,8 +45,17 @@ The app targets shoppers across Australia who are looking for unique and quality
...
+![Shizuka-Kagami](./docs/images/user-pesonas/Sizuka-kagami.png)
+
+On the other hand, Ronnie, a user who has recently become addicted to Japanese food, also faces difficulties in finding Japanese foodstuffs locally. Ronnie is also an engineer. The engineer, Ronnie, also expects the system's ease of use and accessibility to the information required.
```
Comment (ellenpham) [inline README.md]: This sentence, you can add some more info:

`On the other hand, Ronnie, a user who has recently become addicted to Japanese food, also faces difficulties in finding Japanese foodstuffs locally`

--> "On the other hand, Ronnie, a user who has recently become addicted to Japanese food, also faces the difficulty of finding unique and authentic Japanese food products locally. He also prefer a clear product description written in English as he has a hard time choosing the right product when the product labels are mostly written in Japanese."

File: README.md [hunk 3]
```diff
@@ -45,8 +45,17 @@ The app targets shoppers across Australia who are looking for unique and quality
...
+![Ronnie-](./docs/images/user-pesonas/Ronnie-Campbell.png)
+
+
+The final user story focuses on Lara, a system administrator at Sakura Company. She aims to ensure efficient operations and deliver the best user experience. High-quality user and product inventory management are essential to achieve this goal. The system is expected to be intuitive, easy to understand, and provide easy access to crucial functions.
```
Comment (ellenpham) [inline README.md]: maybe change `High-quality user` to `A reliable user database`

File: README.md [hunk 4]
```diff
@@ -45,8 +45,17 @@ The app targets shoppers across Australia who are looking for unique and quality
 - Japanese food products lovers
 
 ## User Stories 
+We construct user stories and personas to improve the online store's user experience for Japanese food products. The process involves identifying and embodying potential users' unique needs and expectations, including a system administrator at Sakura company.
```
Comment (ellenpham) [inline README.md]: Just a few minor spelling and formatting mistakes here and there. I suggest paste the paragraph to Word and spell check first before pasting back to Markdown.

File: docs/images/user-pesonas/Sizuka-kagami.png
Comment (ellenpham) [inline docs/images/user-pesonas/Sizuka-kagami.png:1]: I think the preferred features is missing? `Secure logout for????`

**Comment by Mana12011207** [comment]
Please review user stories again.

**Comment by irenenguyen1017** [comment]
I'm ready to approve your PR. However, I noticed that Ellen has some comments, and I'm not sure if they've been included in your latest version. Please discuss these with Ellen to ensure everything is addressed before merging your PR.

File: docs/images/user-pesonas/Shizuka-Kagami.png
Comment (ellenpham) [inline docs/images/user-pesonas/Shizuka-Kagami.png:1]: There's still a missing bit of "Preferred feature", `Secure logout for ???`

**Review by ellenpham — CHANGES_REQUESTED** [review]
I think there're still missing details in the persona of Shizuka card, please check out comment.

File: docs/images/user-pesonas/Shizuka-Kagami.png
Comment (ellenpham) [reply docs/images/user-pesonas/Shizuka-Kagami.png:1]: Hi Mana, I noticed that this still has not been fixed, can you please check again so we can tick off this task?

**Review by ellenpham — CHANGES_REQUESTED** [review]
Hi Mana, sorry for the back and forth changes but I realized again that the preferred features in the user stories need to be aligned with our features and vice versa. Hopefully this is the last changes.

I comment here so you dont have to go back for the comment in Discord chat history

I think a little changes need to be made for Ronnie and Shizuka user stories:

**Ronnie's motivations:** 

- New discoveries
- Wide range of choices 

(no need delivery because we dont do that feature)

**Ronnie's preferred features:**

- Intuitive platform...
- Simple and effortless...
- **New arrivals section for suggestion of new products** 
- **Search products by name for easily finding particular favorite product**

**Shizukar's preffered features:**
- Secure..
- Saved login...
- Convenient...
- **Order confirmation via email to easily track order as she tends to be forgetful sometimes.** 

Afterwards, you can add Jinny's user story with the below description on top:

"Not everyone know about the website before stumbling on it while browsing the internet, Jinny is one of those people. She is among numerous random shoppers, who are the potential users of the website. As a first-time user, she expects the design and layout of the website to be appealing and memorable, easy and clear navigation to make her first shopping experience effortless and seamless. This would make her come back to shop the next time. Also, she would love to know the popular products with description in English and recommended list of products that go together to gather ideas for her cooking recipes. "

File: README.md [hunk 5]
```diff
@@ -47,8 +47,45 @@ The app targets shoppers across Australia who are looking for unique and quality
...
+We constructed user stories and personas to improve the online store's user experience for Japanese food products. The process involves identifying and embodying customers' unique needs and expectations, including a system administrator at Sakura company.
 
-Working in progress...
+### Initial Market Research
+1. As a busy parent, I want search functionality with a category filtering option to save time searching for the right product.
+
+2. As a busy parent, I want a contact form 24/7 to quickly ask questions or queries about products and services in the online shop, regardless of time or location.
+
+3. As a user who often changes residence, I need to easily update my profile to ensure accurate delivery despite frequent changes in residence.
+
```
Comment (irenenguyen1017) [inline README.md]: This feature has been moved to the 'nice-to-have' list. Currently, we are only providing users with the ability to change their password.

Comment (irenenguyen1017) [reply README.md]: This implies that users are required to input their delivery address each time they make a purchase.

File: docs/images/user-pesonas/Jinny_Chang.png
Comment (irenenguyen1017) [inline docs/images/user-pesonas/Jinny_Chang.png:1]: Just a heads-up, it's not a big deal, but have you noticed that Jinny's salary is getting pretty close to Lara's, and she's in admin?

File: README.md [hunk 6]
```diff
@@ -47,8 +47,45 @@ The app targets shoppers across Australia who are looking for unique and quality
...
+4. As a user, I want the convenience of a Product Search Bar that allows me to find specific pantry items effortlessly. This will save me time and ensure that I quickly locate the products I'm looking for.
+
+5. As a user who loves new discoveries, I want a new arrivals section to give me new suggestions on Japanese products I might enjoy. 
+
+6. As a cooking enthusiast, I want to have an intuitive platform where I can enjoy browsing the different ingredients and products exclusive to Japan.
+
+7. As a non-Japanese user who needs help understanding the Japanese language, I want a website with the products translated into English to understand the product details better.
+
+8. As an administrator, I want the ability to efficiently manage users by viewing their information and assisting with account-related matters.
```
Comment (irenenguyen1017) [inline README.md]: I think we won't go far in assisting with account-related issues, so could you please change this to something like  'As an administrator, I want to capture store analytics data to inform our development strategy, so I can ensure our resources are effectively aligned with business growth.'

Comment (ellenpham) [inline README.md]: We only keep Read operation for admin, so other operations on users including Update and Deactivate, might have to move to nice-to-have.

File: README.md [hunk 7]
```diff
@@ -47,8 +47,45 @@ The app targets shoppers across Australia who are looking for unique and quality
 ![Application-Architecture-Diagram](./docs/images/application-architecture-diagram/application_architecture_diagram.png)
 
 ## User Stories 
+We constructed user stories and personas to improve the online store's user experience for Japanese food products. The process involves identifying and embodying customers' unique needs and expectations, including a system administrator at Sakura company.
 
-Working in progress...
+### Initial Market Research
+1. As a busy parent, I want search functionality with a category filtering option to save time searching for the right product.
+
```
Comment (ellenpham) [inline README.md]: Can you add more user stories for the below features:

- Password recovery
- Site navigation and responsive design
- Shopping cart: allow users to add products to cart, view items in cart and increase, decrease or remove items from cart.
- User functionality to view order history

File: README.md [hunk 8]
```diff
@@ -47,8 +47,45 @@ The app targets shoppers across Australia who are looking for unique and quality
...
+2. As a busy parent, I want a contact form 24/7 to quickly ask questions or queries about products and services in the online shop, regardless of time or location.
+
+3. As a user who often changes residence, I need to easily update my profile to ensure accurate delivery despite frequent changes in residence.
```
Comment (ellenpham) [inline README.md]: This one moves to nice-to-have as we dont have edit user profile for MVP features anymore

File: README.md [hunk 9]
```diff
@@ -47,8 +47,45 @@ The app targets shoppers across Australia who are looking for unique and quality
...
+
+4. As a user, I want the convenience of a Product Search Bar that allows me to find specific pantry items effortlessly. This will save me time and ensure that I quickly locate the products I'm looking for.
```
Comment (ellenpham) [inline README.md]: Maybe change `I want the convenience of a Product Search Bar`

to `I want to be able to search products by keywords, which allows me to ......`

File: README.md [hunk 10]
```diff
@@ -47,8 +47,45 @@ The app targets shoppers across Australia who are looking for unique and quality
...
+
+5. As a user who loves new discoveries, I want a new arrivals section to give me new suggestions on Japanese products I might enjoy. 
+
+6. As a cooking enthusiast, I want to have an intuitive platform where I can enjoy browsing the different ingredients and products exclusive to Japan.
+
+7. As a non-Japanese user who needs help understanding the Japanese language, I want a website with the products translated into English to understand the product details better.
+
+8. As an administrator, I want the ability to efficiently manage users by viewing their information and assisting with account-related matters.
+
+9. As an administrator, the ability to perform CRUD operations on products and select quantities will enable me to manage product inventory efficiently and maintain accurate stock listings.
```
Comment (ellenpham) [inline README.md]: Can you add CRUD operations for product categories as well?

I'm nore sure what do you mean by "select quantities"?

File: README.md [hunk 11]
```diff
@@ -47,8 +47,45 @@ The app targets shoppers across Australia who are looking for unique and quality
...
+
```
Comment (ellenpham) [inline README.md]: Can you add one more user story about the Data Summary for admin dashboard, a summary view of total users, total products, total orders and total revenue?

File: README.md [hunk 12]
```diff
@@ -47,8 +47,45 @@ The app targets shoppers across Australia who are looking for unique and quality
...
+### Follow Up Market (nice to have)
+
+1. As a parent, I need secure logout to prevent accidental purchases from children. 
+2. As a user, I want saved login credentials for easy access.
+3. As a user, I want order confirmation emails to record my order history so I can keep track of stock levels in my pantry. 
+
```
Comment (ellenpham) [inline README.md]: I think this part is missing quite a few nice-to-have features. Can you refer to the project description to add some more?

**Review by irenenguyen1017 — CHANGES_REQUESTED** [review]

**Review by ellenpham — CHANGES_REQUESTED** [review]

**Review by ellenpham — APPROVED** [review]

**Review by irenenguyen1017 — APPROVED** [review]

**Pull request merged**